src/components/Views/ObservationsView/ObservationsView.js

```diff
@@ -98,7 +102,7 @@ class ObservationsView extends React.Component<
                 icons={icons}
               />
             )}
-            data={observations}
+            data={Object.keys(observations).map((key) => observations[key])}
```

Comment:
Probably best memoize this

Reply:
will memoizing cause issues when observations change?

Reply:
nevermind, figured it out

Reply:
this was helpful: https://medium.com/ovrsea/react-re-rendering-purecomponents-and-memoization-553a2cc863c3